MovingBlocks/Terasology
# Work on four more render nodes

### Contains
Refactoring, cleanup and documentation of four render nodes:
- PrePostCompositeNode
- SimpleBlendMaterialsNode
- LightShaftsNode
- InitialPostProcessingNodes

Longer term PrePostComposite and InitialPostProcessing  will see more refactoring work, mostly to split them into smaller, more specialized nodes. 

### How to test
- Run Terasology
- Create a new game or Load an existing one
- verify nothing is wrong with the rendered image

### Notes
I can't see bloom producing any effect, but this seems to be the case also in develop. As I proceed with the refactoring it will become possible and necessary to revamp the debug mode to show the exact contribution of each effect to the final picture - if any. In this case it will allow to verify if the bloom effect is too subtle to notice given the in-game circumstances or if it is completely absent.
